Makefile

```diff
@@ -51,20 +57,15 @@ ifeq ($(shell uname -sm),Darwin arm64)
 	ARCH_PARAM = --arch=amd64
 endif
 
-DEFAULT_IMAGE_TAG = latest
-
 # Semantic versioning (i.e. Major.Minor.Patch)
-is_semantic_version = $(shell [[ $(1) =~ ^[0-9]+\.[0-9]+\.[0-9]+(-.+)?$$ ]] && echo "true")
+VERSION_IS_SEMANTIC = $(shell [[ $(VERSION) =~ ^[0-9]+\.[0-9]+\.[0-9]+(-.+)?$$ ]] && echo "true")
 
 # BUNDLE_VERSION defines the version for the limitador-operator bundle.
 # If the version is not semantic, will use the default one
-bundle_is_semantic := $(call is_semantic_version,$(VERSION))
-ifdef bundle_is_semantic
+ifeq ($(VERSION_IS_SEMANTIC),true)
 BUNDLE_VERSION = $(VERSION)
-IMAGE_TAG = v$(VERSION)
 else
 BUNDLE_VERSION = 0.0.0
-IMAGE_TAG ?= $(DEFAULT_IMAGE_TAG)
```

Comment:
In Authorino Operator, the same change would make the bundle version default to the git sha instead of `latest`: https://github.com/Kuadrant/authorino-operator/blob/96576c28e97511695f8c46870ae3dc5c206219c8/Makefile#L5

Reply:
I think that the default should be "latest", not the current git sha. When the default is the git sha, the bundle needs to change on every change in `main`

Reply:
I agree. I'm just raising awareness so this change won't be ported to Authorino Operator as-is.